docs/platforms/azure.md

```diff
@@ -45,6 +45,7 @@ Once logged in you need to install node.js and Node-RED
 
        curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
        sudo apt-get install -y nodejs build-essential
+       sudo ap-get install npm 
```

Comment:
what is ap-get :-)
This won't work as is - so may as well close...

Reply:
thanks , it's should be 'apt' , (changed now )